daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -825,6 +832,8 @@ message Expr {
     // Extract the given type from Any or return None on type-mismatch ('ExpFromAny')
     // *Available in versions >= 1.dev*
     FromAny from_any = 30;
+
+    TypeConName data_is_serializable = 999;
```

Comment:
What's the semantics of this?

Reply:
```
data Mod:T α₁ ⋯ αₙ = ... is a serializable user-define type
------------------------------------------------------------------------------------------------------
data_is_serializable @Mod:T : ∀ α₁ ⋯ αₙ. Serializable α₁ → ⋯ → Serializable αₙ → Serializable (Mod:T α₁ ⋯ αₙ)
``

Reply:
This is to built proof builtin function for user-defined type.

Reply:
```
data Mod:T α₁ ⋯ αₙ = ... is a serializable user-define type
------------------------------------------------------------------------------------------------------
data_is_serializable : ∀ α₁ ⋯ αₙ. Serializable α₁ → ⋯ → Serializable αₙ → Serializable (Mod:T α₁ ⋯ αₙ)
```

Reply:
It is use to build proof building function for user-defined types

Reply:
I use that to force the type of `(primitive @"BEDataIsSerializable")` to have a type of the good shape when converting to daml-lf (in primitive.hs)

Reply:
So I can’t get equality if I have a phantom type parameter that is not serializable?